kernel/kernel-api/src/test/scala/io/delta/kernel/types/StructTypeSuite.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.types
+
+import org.scalatest.funsuite.AnyFunSuite
+
+class StructTypeSuite extends AnyFunSuite {
```

Comment:
Oh I see. Why not just add the tests in `DataTypeJsonSerDeSuite` itself?

Reply:
Basically, you can roundtrip and test both serialize and deserialize works.

Reply:
you're right. thanks